### PR TITLE
Update COORDINATION.md

### PR DESCRIPTION
**Pull request by ballle98**
Initial set of changes.  Needs more work.

### CODE REVIEW

**Comment by glimchb** [comment]
@ballle98 please fix linters https://github.com/opiproject/opi-prov-life/actions/runs/4295886601/jobs/7486904708

File: COORDINATION.md [hunk 1]
```diff
@@ -6,9 +6,81 @@ Reference [BOOTSEQ.md](./BOOTSEQ.md)
...
 
-## Out-band via BMC
+## Out-band via platform BMC
 
-- i2c
-- ncsi
-- usb
-- others...
+### PLDM State sensors
+Useful State definitions
```
Comment (glimchb) [inline COORDINATION.md:12]: let's add physical layer here... where PLDM can be protocol on top... i2c, usb, others?

File: COORDINATION.md [hunk 2]
```diff
@@ -6,9 +6,81 @@ Reference [BOOTSEQ.md](./BOOTSEQ.md)
 
 - what problems we have today with PCIe, timeouts, errors, retries,...
 
-## Out-band via BMC
+## Out-band via platform BMC
```
Comment (glimchb) [inline COORDINATION.md:9]: let's add a table with definitions, what is "platform BMC"

File: COORDINATION.md [hunk 3]
```diff
@@ -6,9 +6,81 @@ Reference [BOOTSEQ.md](./BOOTSEQ.md)
...
 
-- i2c
-- ncsi
-- usb
-- others...
+### PLDM State sensors
```
Comment (glimchb) [inline COORDINATION.md:11]: let's add diagram to show direction of flow from DPU->BMC on state updates

File: COORDINATION.md [hunk 4]
```diff
@@ -6,9 +6,81 @@ Reference [BOOTSEQ.md](./BOOTSEQ.md)
...
+- https://www.dmtf.org/dsp/DSP0248 DMTF DSP0248 PLDM for Platform Monitoring and Control Specification
+
+### PLDM RDE
+Reference:
+- https://www.dmtf.org/dsp/DSP0218 DMTF DSP0218 PLDM for Redfish Device Enablement
+
+### i2c
+I2C on it's own is not useful unless there is a protocol defined.  
+
+### NC-SI OEM
```
Comment (glimchb) [inline COORDINATION.md:78]: let's mention problem with NCSI

File: COORDINATION.md [hunk 5]
```diff
@@ -6,9 +6,81 @@ Reference [BOOTSEQ.md](./BOOTSEQ.md)
...
+
+### usb
+Could use PLDM State Sensor over PLDM over MTCP over USB
+Reference:
+- https://www.dmtf.org/sites/default/files/standards/documents/DSP0283_0.1.5WIP10.pdf (WIP) DMTF MCTP over USB Binding Specification
```
Comment (glimchb) [inline COORDINATION.md:83]: let's also add an option for ETH over USB... pros/cons... trust establishment... higher level FW vs SW and so on....

**Review by glimchb — APPROVED** [review]
good first start, let's fix linter, few comments inside

**Comment by glimchb** [comment]
It's a good start, I will merge it and we can iterate on this merged content

File: COORDINATION.md (hunk 1 above)
Comment (ballle98) [reply COORDINATION.md:13]: in next PR

**Pull request merged**